src/runner/patch-round.js

```diff
@@ -68,53 +68,36 @@ const roundFractionImpl = {
   },
 };
 
-const roundSignificantImpl = {
-  [DECIMAL_128](arg, significantDigits, mode) {
-    return arg.toSignificantDigits(significantDigits, mode);
-  },
-  [BIG_DECIMAL](arg, significantDigits, mode) {
-    return arg.prec(significantDigits, mode);
-  },
-};
-
-function round(decimal, options = Object.create(null)) {
+const round = (decimal, options = {}) => {
```

Comment:
Just updated this to match the way we are instantiation functions here. It's better to keep it consistent.